src/InsightContainer/index.js

```
@@ -0,0 +1,200 @@
+          item
+          direction="row"
+          alignItems="flex-start"
+          justify="center"
+        >
+          <BlockLoader loading={loading} height="2.5rem">
+            <InsightDataActions
+              title={title}
+              onShare={handleShare}
+              onDownload={handleDownload}
+              onShowData={handleShowData}
+              onCompare={handleComapre}
+            />
+          </BlockLoader>
+        </Grid>
+        <Grid container item className={classes.summaryGrid}>
+          <BlockLoader loading={loading}>
+            <Typography className={classes.summaryHead}>
+              {summaryHead}
+            </Typography>
+            <Typography className={classes.summaryBrief}>
+              {summaryBrief}
+            </Typography>
+          </BlockLoader>
+        </Grid>
+        <Grid container item>
+          <BlockLoader loading={loading}>
+            <div style={{ width: '100%', padding: '0 1.2rem' }}>
+              <A className={classes.analysisLink} href={analysisUrl}>
+                {analysis}
+              </A>
+            </div>
+          </BlockLoader>
+        </Grid>
+      </Grid>
+    </Grid>
+  );
+}
+
+InsightContainer.propTypes = {
+  children: PropTypes.arrayOf(PropTypes.node).isRequired,
+  title: PropTypes.string.isRequired,
+  sourceUrl: PropTypes.string,
+  loading: PropTypes.bool,
+  analysis: PropTypes.string,
+  analysisUrl: PropTypes.string,
+  summaryHead: PropTypes.string,
+  summaryBrief: PropTypes.string,
+  content: PropTypes.shape({
+    width: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    height: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
+  })
+};
+
+InsightContainer.defaultProps = {
+  sourceUrl: undefined,
+  loading: false,
+  analysisUrl: '/profiles/nigeria',
+  analysis: 'Read the country analysis',
+  summaryHead: 'Summary',
+  summaryBrief:
+    'Lorem ipsum dolor sit amec, the related demographic analysis for South Africa',
```

Let remove the lore ipsum, default is undefined and if summary is not provide hide the head and summary.